scripts/flash/list-markets.js

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env node
+
+"use strict";
+
+var chalk = require("chalk");
+var async = require("async");
+
+function listMarketsInternal(augur, universe) {
+  var timestamp = augur.api.Controller.getTimestamp();
+  var currentTime = new Date(timestamp * 1000);
+  augur.markets.getMarkets({ universe: universe, sortBy: "endDate", isSortDescending: true }, function (err, marketIDs) {
+    if (!marketIDs || marketIDs.length === 0) { console.log(chalk.red("No markets available")); process.exit(0);}
+    augur.markets.getMarketsInfo({ marketIDs: marketIDs }, function (err, marketInfos) {
+      if (!marketInfos || !Array.isArray(marketInfos) || !marketInfos.length) return;
+      var infos = marketInfos.sort(function (a, b) { return b.endDate - a.endDate; });
+      async.eachSeries(infos, function (marketInfo, nextMarket) {
```

Comment:
Suggest using an ordinary for loop (or `forEach` etc.) instead of `async.eachSeries`, since nothing in the body of the loop is asynchronous.